bitcoind-rpc-test/src/test/scala/org/bitcoins/rpc/v18/BitcoindV18RpcClientTest.scala

```diff
@@ -0,0 +1,62 @@
+package org.bitcoins.rpc.v18
+import org.bitcoins.rpc.client.common.BitcoindVersion
+import org.bitcoins.rpc.client.v18.BitcoindV18RpcClient
+import org.bitcoins.testkit.rpc.BitcoindRpcTestUtil
+import org.bitcoins.testkit.util.BitcoindRpcTest
+
+import scala.concurrent.Future
+
+class BitcoindV18RpcClientTest extends BitcoindRpcTest {
+  lazy val client: BitcoindV18RpcClient = {
+    new BitcoindV18RpcClient(BitcoindRpcTestUtil.v18Instance())
+  }
+
+  clientAccum.+=(client)
+
+  behavior of "BitcoindV18RpcClient"
+
+  it should "be able to start a V18 bitcoind instance" in {
+
+    client.start().map { _ =>
```

Comment:
And then once you have `clientF: Future[BitcoindV18RpcClient]` above, you should change all of these to ```clientF.map { client => ... }```